src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -185,6 +185,21 @@ const MetaTraderUI = (() => {
                     login   : () => (`${info} (${/demo/.test(accounts_info[acc_type].account_type) ? localize('Demo Account') : localize('Real-Money Account')})`),
                 };
                 $(this).html(typeof mapping[key] === 'function' ? mapping[key]() : info);
+
+                if (key === 'login') {
+                    const mt_financial_company = State.getResponse('landing_company.mt_financial_company');
+                    const mt_gaming_company = State.getResponse('landing_company.mt_gaming_company');
+                    const company = /standard/.test(acc_type) ?
+                                    mt_financial_company.standard
+                                    : /advanced/.test(acc_type) ?
+                                        mt_financial_company.advanced
+                                        : mt_gaming_company.standard;
```

Comment:
please put a bracket around these three lines.
what happens if they add mt_gaming_company.advanced later? do we need to write it in a way that it won't break in that scenario?

Reply:
Noted and addressed in [#8fa5c77](https://github.com/binary-com/binary-static/pull/5006/commits/8fa5c77fa22915f8134b94b78381cc1f9114fe13)